src/components/molecules/closeness-sensing/active.tsx

```diff
@@ -33,9 +34,9 @@ export const Active: FC<{onboarding?: boolean}> = ({onboarding = false}) => {
         {onboarding && (
           <>
             <Spacing s={20} />
-            <Text style={text.default}>
+            <Markdown style={text.default}>
```

Comment:
You don't need `style={text.default}` for the `Markdown` component. Hopefully it still works as it should.